Test/PcapDeviceTest.cs

```diff
@@ -42,7 +69,6 @@ public class PcapDeviceTest
         /// Test that the proper exception is thrown if a user tries to
         /// call GetNextPacket() while a capture loop is running.
         /// </summary>
-        [NonParallelizable]
```

Comment:
this looks like a change that snuck in and is unrelated to this commit

Reply:
There is a `[NonParallelizable]` attribute on the class, so this was redundant.